tests/Cosmos/StakingTests.cpp

```diff
@@ -46,45 +46,11 @@ TEST(CosmosStaking, Staking) {
     ASSERT_EQ("wIvfbCsLRCjzeXXoXTKfHLGXRbAAmUp0O134HVfVc6pfdVNJvvzISMHRUHgYcjsSiFlLyR32heia/yLgMDtIYQ==", signatureInBase64);
 
     auto output = signer.build();
-    
-    ASSERT_EQ(
-        "{"
-            "\"type\":\"auth/StdTx\","
-            "\"value\":{"
-                "\"fee\":{"
-                    "\"amount\":["
-                                    "{\"amount\":\"1018\",\"denom\":\"muon\"}"
-                                "],"
-                    "\"gas\":\"101721\""
-                "},"
-                "\"memo\":\"\","
-                "\"msg\":["
-                        "{"
-                            "\"type\":\"cosmos-sdk/MsgDelegate\","
-                            "\"value\":{"
-                                "\"amount\":{\"amount\":\"10\",\"denom\":\"muon\"},"
-                                "\"delegator_address\":\"cosmos1hsk6jryyqjfhp5dhc55tc9jtckygx0eph6dd02\","
-                                "\"validator_address\":\"cosmosvaloper1zkupr83hrzkn3up5elktzcq3tuft8nxsmwdqgp\""
-                            "}"
-                        "}"
-                    "],"
-                "\"signatures\":["
-                                    "{"
-                                        "\"pub_key\":{"
-                                            "\"type\":\"tendermint/PubKeySecp256k1\","
-                                            "\"value\":\"AlcobsPzfTNVe7uqAAsndErJAjqplnyudaGB0f+R+p3F\""
-                                        "},"
-                                        "\"signature\":\"wIvfbCsLRCjzeXXoXTKfHLGXRbAAmUp0O134HVfVc6pfdVNJvvzISMHRUHgYcjsSiFlLyR32heia/yLgMDtIYQ==\""
-                                    "}"
-                                "]"
-            "}"
-        "}", 
-        output.json());
 
-        ASSERT_EQ("a264747970656a617574682f53746454786576616c7565a463666565a266616d6f756e7481a266616d6f756e7464313031386564656e6f6d646d756f6e6367617366313031373231646d656d6f60636d736781a2647479706576636f736d6f732d73646b2f4d736744656c65676174656576616c7565a366616d6f756e74a266616d6f756e746231306564656e6f6d646d756f6e7164656c656761746f725f61646472657373782d636f736d6f733168736b366a727979716a6668703564686335357463396a74636b7967783065706836646430327176616c696461746f725f616464726573737834636f736d6f7376616c6f706572317a6b757072383368727a6b6e33757035656c6b747a63713374756674386e78736d77647167706a7369676e61747572657381a2677075625f6b6579a26474797065781a74656e6465726d696e742f5075624b6579536563703235366b316576616c7565782c416c636f6273507a66544e56653775714141736e6445724a416a71706c6e79756461474230662b522b703346697369676e61747572657858774976666243734c52436a7a6558586f58544b66484c4758526241416d5570304f313334485666566336706664564e4a76767a49534d485255486759636a735369466c4c79523332686569612f794c674d44744959513d3d", hex(output.encoded()));
```

Comment:
can you keep this assert?

Reply:
@hewigovens that encoding is not needed, json is what their official are using.